Update instructions for old interface that no longer exists

simonkotwicz opened the pull request.

derekantrican commented on Code.gs:
```diff
@@ -6,15 +6,13 @@
-* 3) Install:
-*      New Interface: Make sure your toolbar says "install" to the right of "Debug", then click "Run"
-*      Old Interface: Click "Run" > "Run function" > "install"
+* 2) Settings: Change lines 22-48 to be the settings that you want to use
```
Why were the line numbers changed?

derekantrican reviewed (commented):
Why are these changes suggested? Users can still access the old interface by using "Use classic editor" in the top right (and currently there's still a number of features that haven't been ported to the new interface, so there's still reason to use the old one)

simonkotwicz commented:
Sorry, my bad, I didn't realize the old editor was still available. I updated the PR to just update the line numbers for the personal config and added the new interface instructions for the uninstall function (similar to other places)
